src/presenter/event-point-presenter.js

```diff
@@ -0,0 +1,101 @@
+import EventPointView from '../view/event-point-view.js';
+import EditFormView from '../view/edit-form-view.js';
+import {isPressEscape} from '../utils/common.js';
```

Comment:
Здесь не совсем верный нейминг. Мы проверяем что кнопка escape
```suggestion
import {isEscapeKey} from '../utils/common.js';
```